tests/common.py

```diff
@@ -1069,20 +1069,17 @@ async def start_reauth_flow(
     async def start_reconfigure_flow(
         self,
         hass: HomeAssistant,
-        context: dict[str, Any] | None = None,
-        data: dict[str, Any] | None = None,
+        *,
+        show_advanced_options: bool = False,
```

Comment:
After discussion with @emontnemery on Discord, there shouldn't be any reason to adjust the context - other than setting advanced options.